index.html

```diff
@@ -5,18 +5,19 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link href="styles.css" rel="stylesheet">
+  <script src="script.js" async></script>
   <title>Mix-Or-Match</title>
 </head>
 <body>
   <h1 class="page-title">Mix-Or-Match</h1>
   <div class="overlay-text visible">
     Click to Start
   </div>
-  <div class="game-over-text overlay-text">
+  <div id="game-over-text" class="overlay-text">
```

Comment:
I changed these to ids since it makes the js code easier to use and I am not using them for CSS.